tests/server/api/types/conftest.py

```diff
@@ -42,6 +42,7 @@ def create_context(primary_dataset: Dataset, reference_dataset: Optional[Dataset
             response=None,
             model=create_model_from_datasets(primary_dataset, reference_dataset),
             export_path=Path(TemporaryDirectory().name),
+            db=None,  # TODO: add mock for db
```

Comment:
I'm guessing we can run the in-memory version here?

Reply:
yea, probably. right now it's just stub to satisfy the type checker

Reply:
sorry, not the type checker. it's a required parameter so it won't run unless it has something in it

Reply:
got it